libmdnsd/CMakeLists.txt

```diff
@@ -63,13 +67,33 @@ if(MDNSD_COMPILE_AS_CXX)
 endif()
 
 if(MDNSD_BUILD_OSS_FUZZ)
-    add_library(libmdnsd ${LIBRARY_FILES})
+    add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 else()
-    add_library(libmdnsd SHARED ${LIBRARY_FILES})
+    add_library(libmdnsd SHARED ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
 endif()
 
 if(WIN32 OR MINGW)
   target_link_libraries(libmdnsd wsock32 ws2_32)
 endif()
 
 target_compile_definitions(libmdnsd PRIVATE -DMDNSD_DYNAMIC_LINKING_EXPORT)
+
+include(GNUInstallDirs)
+
+install(
+    TARGETS ${PROJECT_NAME}
+    EXPORT ${PROJECT_NAME}Targets
```

Comment:
Unless you intend to do a `install(EXPORT)` and create a config script, this line does nothing.
See https://cmake.org/cmake/help/latest/command/install.html#export and
https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html

Reply:
True, I snipped that from my personal CMake cook book, were I create ${PROJECT_NAME}Targets.cmake and use it for packaging purposes, but conan-central-index tends to avoid using it.